finalPrep/modZeroHero.js

```diff
@@ -45,40 +63,79 @@
 var announcement = 'Never fear, the Courageous Curly Bracket is here!';
 var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.';
 // assessSituation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
+assessSituation(99, saveTheDay, badExcuse)
 //assessSituation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
+assessSituation(21, saveTheDay, badExcuse)
 //assessSituation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+assessSituation(3,saveTheDay, badExcuse)
 // Declare a new variable - scaryMonster - assigned to an Object with the following key/values
 //   - name (string)
 //   - smell (string)
 //   - weight (integer)
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var address = {
+  number: 34593,
+  street: "34th street NE",
+  state: "NY",
+  zip: 10008,
+  }
+
+var scaryMonster = {
+  name: "Blizzard",
+  smell: "Minty",
+  weight: 32,
+  citiesDestroyed: ["Reykjavik", "Sapporo", "Vorkuta"],
+  luckyNumbers: [7, 12, 72],
+  address: address,
+  }
 
 // Create a new class called SuperHero
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+    this.archNemesis = "The Syntax Error";
+    this.powerLevel = 100;
+    this.energyLevel = 50;
+  }
+  sayName(name){
```

Comment:
With these methods, you have params set up, but they are not really used.  The `name` in your parentheses is not the same as `this.name`, which refers to the attribute in this class. As you look back at this, I don't think having a param for `sayName` or `maximizeEnergy` is really needed, but a param for `gainPower` is a great idea. I may name that param something different than `powerLevel` so you don't get it confused with `this.powerLevel`.